packages/core/src/__tests__/ceramic-anchor.test.ts

```diff
@@ -11,20 +11,34 @@ import { createIPFS, swarmConnect } from './ipfs-util';
 import { TileDoctype } from '@ceramicnetwork/doctype-tile';
 import InMemoryAnchorService from '../anchor/memory/in-memory-anchor-service';
 import { anchorUpdate } from '../state-management/__tests__/anchor-update';
+import ThreeIdResolver from '@ceramicnetwork/3id-did-resolver'
+import KeyDidResolver from 'key-did-resolver'
+import { Resolver } from "did-resolver"
+import { DID } from 'dids'
 
 jest.mock('../store/level-state-store')
 
 const seed = u8a.fromString('6e34b2e1a9624113d81ece8a8a22e6e97f0e145c25c1d4d2d0e62753b4060c83', 'base16')
 
+const makeDID = function(seed: Uint8Array, ceramic: Ceramic): DID {
+  const provider = new Ed25519Provider(seed)
+
+  const keyDidResolver = KeyDidResolver.getResolver()
+  const threeIdResolver = ThreeIdResolver.getResolver(ceramic)
+  const resolver = new Resolver({
+    ...threeIdResolver, ...keyDidResolver,
+  })
+  return new DID({ provider, resolver })
+}
```

Comment:
I guess this could be made into a util somewhere.

Reply:
Yeah agreed.  I wanted to do that here but just wasn't sure the best place to put it to avoid re-introducing dependency cycles.  See this discord thread I just started: https://discord.com/channels/484729862368526356/638777352402239538/828334185658843166